### PR TITLE
recipe for qhub-dask

### PR DESCRIPTION
**Pull request by viniciusdc**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/qhube-dask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask =2.14.0
+    - distributed =2.14
+    - dask-gateway =0.6.1
```
Comment (beckermr) [inline recipes/qhube-dask/meta.yaml:16]: These pins are really restrictive. Are they actually needed?

Comment (viniciusdc) [reply recipes/qhube-dask/meta.yaml:16]: I think so...

Comment (beckermr) [reply recipes/qhube-dask/meta.yaml:16]: Not my package so if you are happy with it, then I am too.

File: recipes/qhube-dask/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+test:
+  imports:
+    - dask
```
Comment (beckermr) [inline recipes/qhube-dask/meta.yaml:20]: change this test to something like `echo 'works'`

Comment (viniciusdc) [reply recipes/qhube-dask/meta.yaml:20]: Thanks, I was looking for something like that :smile:

Comment (beckermr) [reply recipes/qhube-dask/meta.yaml:20]: yep

```
test:
  commands:
    - echo 'works'
```

**Pull request merged**